js/views/reader_view.js

```diff
@@ -730,11 +726,6 @@ ReadiumSDK.Views.ReaderView = function(options) {
     function openPage(pageRequest, dir) {
 
         initViewForItem(pageRequest.spineItem, function(isViewChanged){
-
-            if(!isViewChanged) {
-                _currentView.setViewSettings(_viewerSettings);
-            }
-
```

Comment:
Isn't the removal of the call to `setViewSettings()` on the `_currentView` instance going to create problems in cases when the view type has *not* changed? See:

```javascript
    function initViewForItem(spineItem, callback) {

        var desiredViewType = deduceDesiredViewType(spineItem);

        if(_currentView) {

            if(self.getCurrentViewType() == desiredViewType) {
                callback(false); // FALSE means that setViewSettings() will be called on _currentView (except for the handleViewportResize() case)
                return;
            }

...


        _currentView.setViewSettings(_viewerSettings);

            callback(true); // TRUE means that setViewSettings() will NOT be called on _currentView
...
```